storybook/serve.json

```diff
@@ -0,0 +1,6 @@
+{
+  "public": "./storybook-static",
+  "cleanUrls": false,
+  "directoryListing": false,
+  "redirects": [{ "source": "/", "destination": "/index.html" }]
+}
```

Comment:
This file is used by the `serve` command of the package.json. It configures the server to be able to serve the production-build of storybook locally in a comparable fashion to what is currently live.